README.md

```diff
@@ -50,8 +46,53 @@ sbt clean riffRaffUpload
 
 The `riffRaffUpload` task will execute the tests in your project, and then upload the artifacts and manifest.
 
-Customisation
--------------
+### Within GitHub Actions
+
+The `riffRaffUpload` sbt task will upload files to S3. When run in TeamCity, we gain credentials via TeamCity's `InstanceProfile` policy.
+
+To give GitHub Actions permissions to upload to S3 use the [`@guardian/actions-assume-aws-role` Action](https://github.com/guardian/actions-assume-aws-role).
+Ensure you use the Action before the `sbt clean riffRaffUpload`.
+A secret (`GU_RIFF_RAFF_ROLE_ARN`) has been added Guardian GitHub organisation that can be used for the value of `awsRoleToAssume`.
+
+For example:
+```yaml
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: guardian/actions-assume-aws-role@v1
+        with:
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - run: sbt clean riffRaffUpload
+```
+
+#### Migrating Riff-Raff project to GitHub Actions from TeamCity
+Riff-Raff will only continuous deploy builds of the default branch if the latest build number is greater than the previously deployed build number.
+
+If you've been running in TeamCity for a while you'll likely have a pretty large build number.
+When moving to GitHub Actions, the build number restarts from 1.
+Therefore, you'll likely witness your project's continuous deployment no longer working.
+
+To solve this, it is easiest to customise the values of `riffRaffManifestProjectName` and `riffRaffPackageName`, creating a new project in Riff-Raff and then add a restriction to the previous project.
```

Comment:
```suggestion
To solve this, it is easiest to customise the values of `riffRaffManifestProjectName` and `riffRaffPackageName`, creating a new project in Riff-Raff. Once this change has been merged, you should also:

 1. Add a [restriction](https://riffraff.gutools.co.uk/deployment/restrictions/new) which prevents anyone from accidentally deploying the old Riff-Raff project.
 2. Update your [Continuous Deployment configuration](https://riffraff.gutools.co.uk/deployment/continuous) to use the new project name. 
 3. Update your [Scheduled Deployment configuration](https://riffraff.gutools.co.uk/deployment/schedule) to use the new project name.
 4. Pause the TeamCity project - this can be deleted entirely once you are confident that the GitHub Actions pipeline is working
```